volunteer.markdown

```diff
@@ -13,24 +13,27 @@ layout: page
 ## Get in touch and talk to us
 Join the conversation on [Slack](http://slack.hotosm.org/)! A large number of volunteers and staff discuss mapping, emergencies, Working Groups and other matters on our Slack channel.
 
-## Mapping Volunteers
+<span class="anchor" id="mapping-volunteers"></span>
+<h2>Mapping Volunteers</h2>
 
 As a mapping volunteer, you can get involved with projects around the world, and contribute to humanitarian mapping through OpenStreetMap. Even if you are a new mapper, anyone can use a computer to create basemaps by drawing over buildings and roads, or use a phone to swipe and tap population hotspots in the MapSwipe app. If you have never mapped before and would like to help, visit the Tasking Manager's [Learn page](https://tasks.hotosm.org/learn) to get started mapping in less than 30 minutes.
 
 <a href="https://tasks.hotosm.org/learn" class="btn btn-primary btn-lg btn-chevron">Learn how to map</a>
 
 Please also join the discussion on Slack (see above).
 
-## Humanitarian and GIS Professionals
+<span class="anchor" id="humanitarian-and-gis-professionals"></span>
```

Comment:
Please move this to https://github.com/hotosm/hotosm-website/blob/gh-pages/_layouts/get-involved.html#L18-L20.